src/services/0x.js

```diff
@@ -0,0 +1,195 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import axios from 'axios';
+import { BigNumber as EthersBigNumber } from 'ethers';
+import { reportErrorLog } from 'utils/common';
+
+// based on https://github.com/Rari-Capital/rari-dApp/blob/6d07a47206f409ee6c52690f43c172c25a26071b/src/rari-sdk/0x.js
+export const get0xSwapOrders = async (
+  inputTokenAddress: string,
+  outputTokenAddress: string,
+  maxInputAmountBN: EthersBigNumber,
+  maxMakerAssetFillAmountBN?: EthersBigNumber,
+) => {
+  const { data: decoded } = await axios.get(
+    `https://api.0x.org/swap/v0/quote?sellToken=${
```

Comment:
Wdyt if making this little more readable? Like contructing the url before the call 👀